bin/fin

```diff
@@ -6176,6 +6182,10 @@ load_configuration ()
 	# https://docs.docker.com/compose/reference/envvars/#compose_convert_windows_paths
 	export COMPOSE_CONVERT_WINDOWS_PATHS=1
 
+	# get cli image version
+	CLI_IMAGE_VER=$(docker-compose --verbose config --hash="cli" 2>&1 | grep 'docker inspect_image <-' | cut -d"'" -f2 | sed 's#/#-#g;s#:#-#g')
```

Comment:
This adds almost 2s to `load_configuration()`, which is used in many places in fin.
So this would slow everything down substantially. We need to figure out a different approach.